crates/meroctl/src/cli/context.rs

```diff
@@ -32,13 +36,13 @@ pub struct ContextCommand {
 #[derive(Debug, Subcommand)]
 pub enum ContextSubCommands {
     #[command(alias = "ls")]
-    List(list::ListCommand),
-    Create(create::CreateCommand),
-    Join(join::JoinCommand),
+    List(ListCommand),
+    Create(CreateCommand),
+    Join(JoinCommand),
 }
 
 impl ContextCommand {
-    pub async fn run(self, args: RootArgs) -> eyre::Result<()> {
+    pub async fn run(self, args: RootArgs) -> EyreResult<()> {
```

Comment:
oof, I.. do not like this one..

Reply:
There is a reason 🙂 

The main reason is consistency - if we are applying this naming approach to all types then it unifies things that are named generically `Error`, `Result`, and the like, and prevents issues where people may otherwise `use eyre::Result` which would then silently override the standard `Result`. For such cases it's a good pattern to follow.

Reply:
> and prevents issues where people may otherwise use eyre::Result which would then silently override the standard Result

and how is it "prevent"ed?

Reply:
It's prevented because it's not overriding the standard `Result`.

Reply:
Yet, nothing stops you from overwriting the standard result.

Especially with the single-identifier enforcement.

You'd have to know about the alias of `EyreResult` elsewhere to do the same.

While you're not allowed to use `eyre::Result`, there's nothing to guide you to the appropriate conventional use.